File: leetcode/palindrome_num/str.py
def is_palindrome(x):
	x = str(x)
	l = len(x)
	for i in range(l // 2):
		if x[i] != x[l - i - 1]:
			return False
	return True

from sys import argv
if is_palindrome(int(argv[1])):
	print("t")
else:
	print("f")

File: leetcode/palindrome_num/str.js
/**
 * @param {number} x
 * @return {boolean}
 */
function isPalindrome(x) {
	x = String(x);
	var h = x.length / 2;
	for (var i = 0; i < h; i++) {
		if (x[i] !== x[x.length - i - 1]) return false;
	}
	return true;
}

const { argv } = require('process');
console.log(isPalindrome(Number(argv[2])) ? 't' : 'f');
